flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -2584,14 +2591,17 @@ private boolean validateSequence(
         public static volatile boolean killedLeaderBefore;
         public static volatile boolean hasBeenCheckpointedBeforeFailure;
 
+        private static KafkaTestEnvironment kafkaServerToKill;
```

Comment:
I tend to use `kafkaServerToShutdown` name because we call command `shutdown` instead of `kill ` command